Userland/Libraries/LibGUI/ComboBox.cpp

```diff
@@ -113,6 +113,7 @@ ComboBox::ComboBox()
     };
 
     m_list_window = add<Window>(window());
+    m_list_window->set_window_type(GUI::WindowType::Tooltip);
```

Comment:
This seems extremely brittle

Reply:
I figured I'd do the same thing like `AutocompleteProvider`. Do you think adding a dedicated window type for this (and probably also `AutocompleteProvider` plus some other window in the `Locator` class) makes sense? How about `Popup` or `TemporaryPopup`?

Reply:
if it's architectured as a window, I think having a distinct window type would make a lot of sense, yep :)